run_page/gpxtrackposter/track.py

```diff
@@ -334,11 +342,10 @@ def to_namedtuple(self, run_from="gpx"):
         d = {
             "id": self.run_id,
             "name": (
-                f"run from {run_from} by {self.device}"
-                if self.device
-                else f"run from {run_from}"
+                self.track_name if self.track_name else "none"
```

Comment:
> For the default value, can we use "" instead of "none"?

no problem

Reply:
> Additionally, for names already synced in the db, please include instructions in the readme on enabling the feature for all data, possibly with a SQL, to reset names to default.

It works if I remove all the caches and regenerate the running page completely. I am trying resolving problems on updating synced activities.

Reply:
> For the default value, can we use "" instead of "none"?

resolved in https://github.com/yihong0618/running_page/pull/733/commits/d9ae3aca69fb5131de0f02d87fab36066722b48e